channels/google/accounts/account-setup.rst

```diff
@@ -0,0 +1,123 @@
+Account Setup
+=============
+
+After you’ve :doc:`registered Zammad as an OAuth app <register-app>`
+in your Google Developer settings,
+you can begin connecting Gmail accounts to Zammad.
+
+☠️ **But first, a word of warning!** The import process does things you might not expect:
+
+.. danger:: 🚯 **Zammad will delete all emails in your inbox
+   during the import process.**
```

Comment:
As you can see, I put this warning back. It's good that the option is not hidden as an "expert" setting in Google channels, but it's still VERY dangerous default behavior, so I think all users should be alerted to it.

Reply:
If you think it's still important enough to outline, we should keep it. 🙌